src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerProgressPaperCase.java

```diff
@@ -47,7 +48,8 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
             .showSummary()
             .aboutToSubmitCallback(this::aboutToSubmit)
             .grant(CREATE_READ_UPDATE,
-                CASE_WORKER)
+                CASE_WORKER,
+                SYSTEMUPDATE)
```

Comment:
do we need system update for events? System update is a background user so they don't login using UI ?